Userland/Libraries/LibWeb/CSS/StyleProperties.cpp

```diff
@@ -263,6 +263,14 @@ float StyleProperties::flex_shrink() const
     return value.value()->to_number();
 }
 
+int StyleProperties::order() const
+{
+    auto value = property(CSS::PropertyID::Order);
+    if (!value.has_value() || !value.value()->has_number())
+        return 0;
+    return value.value()->to_number();
```

Comment:
As with Properties.json, this code wants to be working with integers - `has_integer()` and `to_integer()`.